packages/browser/src/sessionInfo/SessionInfoManager.ts

```diff
@@ -165,10 +172,21 @@ export class SessionInfoManager implements ISessionInfoManager {
         secure: true,
       }
     );
+
     const issuer = await this.storageUtility.getForUser(sessionId, "issuer", {
-      secure: true,
+      secure: false,
     });
 
+    if (
+      clientId === undefined &&
+      idToken === undefined &&
+      isLoggedIn === undefined &&
+      webId === undefined &&
+      refreshToken === undefined
```

Comment:
Is it true now that a `refreshToken` is **_always_** going to be stored...?

Reply:
No, to the contrary: we just return `undefined` if none of these values are stored, but if at least one of them is defined, then we return whatever we have. In the browser code, it's unlikely that we have a refresh token at all, and this is going to be removed when we clean up the code to remove support for them, but right now it doesn't hurt to check for it.

Reply:
D'oh - yeah, of course - sorry. Yeah, it was the 'non-browser-ness' of the Refresh Token that piqued my attention here, but I mistakenly read the logic there as being `||` instead of `&&` for some reason.